lib/sinon/stub-non-function-property.js

```diff
@@ -3,15 +3,13 @@
 var valueToString = require("./util/core/value-to-string");
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 
-function stubNonFunctionProperty(object, property, value) {
+function stubNonFunctionProperty(object, property) {
     var original = object[property];
 
     if (!hasOwnProperty.call(object, property)) {
         throw new TypeError("Cannot stub non-existent own property " + valueToString(property));
     }
 
-    object[property] = value;
```

Comment:
Stubbing properties is the responsibility of `collection.js`, it has [several tests covering this](https://github.com/sinonjs/sinon/blob/5eca462a5f769c7de1622886be66d76a67f245a1/test/collection-test.js#L131-L182).